cmd/prometheus/main.go

```diff
@@ -46,6 +46,24 @@ func main() {
 	os.Exit(Main())
 }
 
+var (
+	configAttempts = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "prometheus",
+		Name:      "configuration_attempts",
+		Help:      "number of times a configuration reload was attempted.",
+	})
+	configSuccesses = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "prometheus",
+		Name:      "configuration_successes",
+		Help:      "number of times a configuration reload was successful.",
+	})
+	configSuccessTime = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "prometheus",
+		Name:      "configuration_last_success",
+		Help:      "Timestamp of the last successful configuration reload.",
+	})
+)
+
```

Comment:
> So maybe a boolean 1 / 0 gauge like config_last_reload_successful would make sense? Then you can alert if that gauge stays 0 for too long.

Yes, that's what I was generally thinking (and having that also on the status page as a big warning with the last error would be good).